src/index.ts

```diff
@@ -20,11 +20,18 @@ const client = new CookiecordClient(
 		prefix: ['!', 't!'],
 	},
 	{
-		ws: { intents: Intents.NON_PRIVILEGED },
 		partials: ['REACTION', 'MESSAGE', 'USER', 'CHANNEL'],
-		disableMentions: 'everyone',
+		allowedMentions: {
+			parse: ['users'],
+		},
```

Comment:
Not sure what this should be set to; strangely this doesn't seem to block @everyone pings?

Reply:
Maybe you need `roles: []`? I'm not too sure and this is a bit scary since I also have this in a different bot I run.